### ISSUE TITLE
Weird logic in beastmaster.java

### ISSUE THREAD

**Issue by PlNG** [NerdNu/BeastMaster#3]
https://github.com/NerdNu/BeastMaster/blob/b3f5d39460128238ecf42a5d6e0a929c061c99a9/src/nu/nerd/beastmaster/BeastMaster.java#L743

https://github.com/NerdNu/BeastMaster/blob/b3f5d39460128238ecf42a5d6e0a929c061c99a9/src/nu/nerd/beastmaster/BeastMaster.java#L745

If I understand the code correctly, the way the current code is set up, the first will always be false and the second will always be true because attackingplayer or attackingmob gets set, but the whole block is on the premise that attackingmob is set at 729. I think that the "ignore infighting" logic was supposed to be closed with brackets at line 737.

**Comment by totemo** [comment]
Good catch. I clearly got a bit lost last time I was working on that. I'll fix it up on the next maintenance. Soon.
